### DISCUSSION TITLE
[文章排版] 在notion中代码块和数字列表结合后列表会重新排列

### DISCUSSION THREAD

**Discussion by Ericjsz** [tangly1024/NotionNext#1547] [Q&A]
如图notion
![image](https://github.com/tangly1024/NotionNext/assets/37693244/a751e6c0-a854-406c-b5ba-44ccacd685f3)
在网页上是
![image](https://github.com/tangly1024/NotionNext/assets/37693244/4dce5191-8ecb-41ad-894f-fdaba0acb3bb)

在notion上编写技术文章, 将代码块插入到顺序的数字列表中, 顺序是正常的从小到大排列。等部署的时候, 排版则是重新从插入的代码块开始计算, 也就是数字列表的第一位。请问有人遇到过这样的问题吗? 欢迎讨论一下

**Comment by wpttt** [comment]
给代码快加一层缩进时势
